lib/open_api_spex/paths.ex

```diff
@@ -25,8 +25,14 @@ defmodule OpenApiSpex.Paths do
 
   @doc """
   Create a Paths map from a list of routes.
+
+  ## Example
+
+      Paths.from_routes([
+        %{path: "/v1/contacts", verb: :post, plug: MyAppWeb.V1.ContactController, plug_opts: :create}
+      ])
   """
-  @spec from_routes([%{path: String.t(), verb: atom()}]) :: t
+  @spec from_routes([%{path: String.t(), verb: atom(), plug: module(), plug_opts: atom()}]) :: t
```

Comment:
There is a type we can use here:

```suggestion
  @spec from_routes([PathItem.route()]) :: t
```

Reply:
so isn’t the path needed after all?

Reply:
it groups the routes by path in this function, so it’s needed

Reply:
You're right, the routes is a list of maps like:

```elixir
[
  %{
    helper: "render_spec",
    metadata: %{log: :debug},
    path: "/docs/api_spec",
    plug: OpenApiSpex.Plug.RenderSpec,
    plug_opts: [],
    verb: :get
  },
  %{
    helper: "swagger_ui",
    metadata: %{log: :debug},
    path: "/docs/api",
    plug: OpenApiSpex.Plug.SwaggerUI,
    plug_opts: [path: "/docs/api_spec"],
    verb: :get
  }
]
```

and they do have a `:path` key.

I think we should add it to the `route` type.

```suggestion
  @type route ::
          %{path: String.t(), verb: atom, plug: atom, opts: any}
          | %{path: String.t(), verb: atom, plug: atom, plug_opts: any}
```

It's clear from the [existing tests](https://github.com/open-api-spex/open_api_spex/blob/master/test/path_item_test.exs#L10) that routes are expected to have `path`.

Reply:
I've applied the changes, could you please review it again?